src/mono/browser/runtime/package-lock.json

```diff
@@ -1,8 +1,2756 @@
 {
   "name": "@microsoft/dotnet-runtime",
```

Comment:
Are the changes in this file intentional - why do we need them?

Reply:
No, reverted.